### ISSUE TITLE
Downloading Module is failing

### ISSUE THREAD

**Issue by sunil-saini** [phalaaxx/milter#24]
Installing of module is failing 

```
github.com/phalaaxx/milter: github.com/phalaaxx/milter@v0.0.0-20240411080318-dd6508ad3a76: parsing go.mod:
module declares its path as: milter
        but was required as: github.com/phalaaxx/milter
```

same is there with go get

```
$ go get github.com/phalaaxx/milter
go: github.com/phalaaxx/milter@v0.0.0-20240411080318-dd6508ad3a76: parsing go.mod:
        module declares its path as: milter
                but was required as: github.com/phalaaxx/milter
```

This is happening due to milter having its path as `milter` instead of `github.com/phalaaxx/milter` in [go.mod](https://github.com/phalaaxx/milter/blob/master/go.mod)

**Comment by phalaaxx** [comment]
Ah sorry, I just now see your issue. However I fixed that on August 16th so it's all OK now, thanks for the report.

**Comment by phalaaxx** [comment]
It's fixed in 8304fd4788c2f7a7118ec8859b3c7cf8a2590ee5.
